integration-tests/testdb.go

```
@@ -0,0 +1,148 @@
+		Database:  dbName,
+		EnableSsl: false,
+	}
+
+	adminConnStr := "postgresql://postgres:postgres@localhost:5440/postgres?sslmode=disable"
+	adminDb, err := sql.Open("postgres", adminConnStr)
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to postgres admin: %w", err)
+	}
+	defer adminDb.Close()
+
+	if err := adminDb.Ping(); err != nil {
```

golangci-lint (2.11.4)

[error] (*database/sql.DB).Ping must not be called. use (*database/sql.DB).PingContext

(noctx)